docs/_config.yml

```diff
@@ -21,7 +21,7 @@ github_info:
 
 # Uncomment to enable AnchorJS functionality on docs layout.
 # See https://github.com/bryanbraun/anchorjs for more information.
-anchor_js_targets: [h1, h2, h3, h4, h5, h6]
+#anchor_js_targets: [h1, h2, h3, h4, h5, h6]
```

Comment:
IIRC, that enables direct links like 

https://crowdstrike.github.io/community/open-source-policy/#everyone-else

This seems like a nice feature.

Reply:
interesting.... cause it causes some errors on jekyll builds... at least when I was doing it locally.

Reply:
```
% bundle exec jekyll serve
Configuration file: /Users/swells/Documents/src/github/CrowdStrike/community/docs/_config.yml
            Source: /Users/swells/Documents/src/github/CrowdStrike/community/docs
       Destination: /Users/swells/Documents/src/github/CrowdStrike/community/docs/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
                    done in 4.582 seconds.
 Auto-regeneration: enabled for '/Users/swells/Documents/src/github/CrowdStrike/community/docs'
    Server address: http://127.0.0.1:4000
  Server running... press ctrl-c to stop.
```

or

```
% bundle exec jekyll build
Configuration file: /Users/swells/Documents/src/github/CrowdStrike/community/docs/_config.yml
      Source: /Users/swells/Documents/src/github/CrowdStrike/community/docs
    Destination: /Users/swells/Documents/src/github/CrowdStrike/community/docs/_site
 Incremental build: disabled. Enable with --incremental
   Generating... 
          done in 4.502 seconds.
 Auto-regeneration: disabled. Use --watch to enable.```

Reply:
odd... undid the change